lua/conform/formatters/markdown-toc.lua

```diff
@@ -0,0 +1,15 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/jonschlinkert/markdown-toc",
+    description = "API and CLI for generating a markdown TOC (table of contents) for a README or any markdown files.",
+  },
+  command = "markdown-toc",
+  stdin = false,
+  args = function()
+    -- use the indentation set in the current buffer, effectively allowing us to
+    -- use values from .editorconfig
+    local indent = vim.bo.expandtab and (" "):rep(vim.bo.tabstop) or "\t"
```

Comment:
nit: technically formatting can happen on the non-current buffer (for example, if you're in another file and `:wa`). To handle this, use `funtion(ctx)` and `vim.bo[ctx.buf]`